pkgs/development/libraries/libhttpseverywhere/default.nix

```diff
@@ -1,20 +1,22 @@
 {stdenv, fetchurl, gnome3, glib, json_glib, libxml2, libarchive, libsoup, gobjectIntrospection, meson, ninja, pkgconfig,  valadoc}:
 
 stdenv.mkDerivation rec {
-  major = "0.4";
-  minor = "5";
+  major = "0.6";
+  minor = "1";
   version = "${major}.${minor}";
 
   name = "libhttpseverywhere-${version}";
 
   src = fetchurl {
     url = "mirror://gnome/sources/libhttpseverywhere/${major}/libhttpseverywhere-${version}.tar.xz";
-    sha256 = "07sgcw285rl9wqr5k7srs3fj7fhgrrw6w780jx8wy8jw2bfwlvj2";
+    sha256 = "0pwnh5mbbcb9hpx7by5f5j5qj7kchflwx7dzh49hbkw1jvwrx655";
   };
 
   nativeBuildInputs = [ gnome3.vala valadoc  gobjectIntrospection meson ninja pkgconfig ];
   buildInputs = [ glib gnome3.libgee libxml2 json_glib libsoup libarchive ];
 
+  mesonFlags = "-Denable_valadoc=true";
```

Comment:
It would be better to move the documentation to `devdoc` output but it would require patching [`meson.build`](https://github.com/GNOME/libhttpseverywhere/blob/0.6.1/meson.build#L83)

Reply:
Is this a convention of nixpkgs? If so, I didn't know. Patched the build file.